index.html

```diff
@@ -1011,6 +1011,33 @@ <h4>Exchange Examples</h4>
     <h2>Privacy Considerations</h2>
     <p>
     </p>
+
+    <section>
+      <h3>Phone Home Considered Harmful</h3>
+
+      <p>
+It is considered a bad practice for a <a>verifier</a> to contact an
+<a>issuer</a> about a specific <a>verifiable credential</a>. This 
+practice is known as "phoning home" and can result in a mismatch 
+in privacy expectations between <a>holders</a>, <a>issuers</a>,
+<a>verifiers</a>, etc., as it allows <a>issuers</a> to correlate
+<a>holders</a> with <a>verifiable credentials</a> without their
```

Comment:
```suggestion
<a>holders</a> with <a>verifiable presentations</a> to <verifiers</a> without their
```

Reply:
I don't think `correlate <a>holders</a> with <a>verifiable presentations</a> to <verifiers</a>` is accurate, because the `issuer` can't correlate `verifiable presentations` at all, because — even though the `verifier` only receives `verifiable presentations` of `verifiable credentials` — the `issuer` should never receive any `verifiable presentation` identifiers, only `verifiable credential` identifiers.